### DISCUSSION TITLE
버전관리 관련 제안사항

### DISCUSSION THREAD

**Discussion by chanhihi** [jiphyeonjeon-42/frontend#368] [Ideas]
깃 태그를 사용해서 버전관리하는데 있어, 
시맨틱 버전 ([Semantic Versioning](https://semver.org/lang/ko/))  을 사용해서 했으면 좋겠습니다.

**Comment by scarf005** [comment]
시맨틱 버전을 사용하게 된다면 [시맨틱 릴리즈(semantic-release)](https://github.com/semantic-release/semantic-release)를 사용하여 커밋 메시지를 통해 버전을 자동 관리하는 것도 좋은 방법이 될 것 같습니다.
